contrib/babelfishpg_tsql/src/tablecmds.c

```diff
@@ -115,12 +115,6 @@ void pre_check_trigger_schema(List *object, bool missing_ok){
 				(errcode(ERRCODE_UNDEFINED_OBJECT),
 				errmsg("trigger \"%s.%s\" does not exist",
 						trigger_schema ,depname)));
-			} else if (list_length(object) == 1 && strcasecmp(schema_name, get_dbo_schema_name(get_cur_db_name())) != 0
```

Comment:
why is this check removed?

Reply:
Becuase if pg find the same trigger name but with different dbo schema name doesn't really mean the trigger doesn't exit at this time, the trigger might be found later. The error can only be thrown after checking all triggers.